### DISCUSSION TITLE
Problems with Charging Power - only 85W

### DISCUSSION THREAD

**Discussion by oehrn** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1293] [General]
I have the Pro version of AlDente, and I'm facing an issue where my MacBook only charges at a maximum of 85W, even though it should be able to reach up to 140W.
When I restart my MacBook without AlDente in the startup, it reaches the expected power level, but as soon as I start the program, the charging power drops significantly.

Is there a workaround or a solution for this?

Greetings
Max

**Comment by MatthiasKerbl** [comment]
Hi @maxxt98,

AlDente does not influence the charging power. Is it possible that the normal charging power dropoff during the charging process coincided with AlDente being started?

Best,
Matthias

**Comment by oehrn** [comment]
Hello Matthias,

The MacBook is at 20%. After a restart without AlDente, I'm charging with 95W. As soon as I start AlDente (at around 23%), the charging power drops to below 30W and then builds back up to 85W.

Greetings
Max

**Reply by MatthiasKerbl** [reply]
> Hello Matthias, The MacBook is at 20%. After a restart without AlDente, I'm charging with 95W. As soon as I start AlDente (at around 23%), the charging power drops to below 30W and then builds back up to 85W. Greetings Max Am 13.08.2024 um 09:17 schrieb Matthias Kerbl ***@***.***>: ﻿ Hi @maxxt98<https://github.com/maxxt98>, AlDente does not influence the charging power. Is it possible that the normal charging power dropoff during the charging process coincided with AlDente being started? Best, Matthias — Reply to this email directly, view it on GitHub<[#1293 (comment)](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/discussions/1293#discussioncomment-10322269)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANWIH2QZ7NC343B76PUVOVTZRGXJJAVCNFSM6AAAAABMNSGUNWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZSGIZDMOI>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Hi @maxxt98,

It sounds like charging is paused for a moment. Maybe due to Heat Protection, Sailing Mode, or Charge Limiter. Did you check the Power Flow chart to see where to energy is coming from and going to?

Best,
Matthias

**Reply by oehrn** [reply]
Hi Matthias,
I was about to record a video to show exactly how the performance changes, but I couldn't reproduce the error. I’ll keep monitoring the behavior.

Greetings
Max
